modules/statics/test/unit/coins.ts

```diff
@@ -35,3 +38,10 @@ describe('ERC20 Coins', () => {
       }, {});
   });
 });
+
+describe('All Coins', () => {
```

Comment:
nice!

Reply:
`statics` should not have any outside dependencies

when we add a coin, we should add it in `statics` first and later use these definitions in downstream projects

this test creates a circular dependency